Doc/extending/embedding.rst

```diff
@@ -275,6 +275,91 @@ will depend on the details of the C++ system used; in general you will need to
 write the main program in C++, and use the C++ compiler to compile and link your
 program.  There is no need to recompile Python itself using C++.
 
+.. _freezingandembeddingmanually:
+
+Freezing Modules in Manually Embedded Python
+============================================
+
+While it is possible to use frozen modules in embedded python if the main module
+is also frozen. It seems people are bitten when they don't have their main module
```

Comment:
What do you mean by 'bitten'? I think it's better to use direct language and be clear on what the problem might be.

Reply:
As in when they try to run it, it won't work properly.

Reply:
Is there a specific exception/error that appears, or does it do something like hang?

Reply:
I believe it crashes yeah